cmd/influxd/upgrade/upgrade_test.go

```diff
@@ -0,0 +1,69 @@
+package upgrade
+
+import (
+	"io/ioutil"
+	"os"
+	"path/filepath"
+	"testing"
+
+	"github.com/docker/docker/pkg/testutil/assert"
```

Comment:
To avoid the proliferation of differing `assert` packages, please use `stretchr/testify/assert`

```suggestion
	"github.com/stretchr/testify/assert"
```

Reply:
This, accidental import, no intention :(